script/main.js

```diff
@@ -116,4 +116,20 @@ function list(){
     listDiv.appendChild(task);
     
   });
-}
\ No newline at end of file
+}
+
+let countDone = 0;
+let countUnDone = 0;
```

Comment:
when i open the page for the first time, the counting is not 0, change the default count
![Screenshot from 2021-07-18 14-35-18](https://user-images.githubusercontent.com/47992412/126065502-97a7f318-9c6c-4af8-b728-bbbcd25723c0.png)

Reply:
I just forget to change it 
now is done